docs/python/modelspec/programmatic/mechanisms/nmodl2.rst

```diff
@@ -381,18 +381,22 @@ Description:
     ``TODO``: Add description and existing example mod file
 
 
-BEFORE
-~~~~~~
+BEFORE / AFTER
+~~~~~~~~~~~~~~
 
 Description:
-    ``TODO``: Add description and existing example mod file
-
-
-AFTER
-~~~~~
+    BEFORE INITIAL executes just before any INITIAL blocks of any mod file execute (but after all the FInitializeHandler(0,..) are called).
```

Comment:
```suggestion
    The `BEFORE INITIAL` block executes just before any `INITIAL` blocks of all mod files (but after all the FInitializeHandler(0,..) are called).
```

Reply:
not sure how this renders.. shouldn't the ` be a `` - also, maybe you need to make this a bullet list, otherwise it is all connected into one paragraph.